cmd/apprepository-controller/controller.go

```diff
@@ -69,6 +70,8 @@ const (
 	// MessageResourceSynced is the message used for an Event fired when an
 	// AppRepsitory is synced successfully
 	MessageResourceSynced = "AppRepository synced successfully"
+	//DefaultLifeTimeTTL max sync lifetime job //https://kubernetes.io/docs/concepts/workloads/controllers/job/#clean-up-finished-jobs-automatically
+	DefaultLifeTimeTTL = 3600
```

Comment:
I'm not sure why we need this constant here, since it already defaults to "3600" from the command-line arg default that you've set? If you agree, perhaps just delete this const and move the helpful comment to the main.go?